packages/x-charts-pro/src/FunnelChart/funnelSectionClasses.ts

```diff
@@ -42,7 +43,17 @@ export const useUtilityClasses = (props: FunnelSectionProps) => {
   return composeClasses(slots, getFunnelSectionUtilityClass, classes);
 };
 
+export const useLabelUtilityClasses = (props: FunnelSectionLabelProps) => {
+  const { classes, seriesId, dataIndex } = props;
+
+  const slots = {
+    label: ['label', `series-${seriesId}`, `data-index-${dataIndex}`],
```

Comment:
Could we avoid creating the `series-` and `data-index-` classes? Otherwise it's more breaking changes that we'll need to do in the next major. We should be able to achieve the same with data attributes.

Reply:
We can, but we will have a lot of breaking changes in v9 related to this. I would rather we have consistent APIs from the start though

Reply:
I agree we could start deprecating (I would create a separate PR to deprecate everything in one go), but I don't agree we should stop adding the classes in new components.

My point regarding consistency is consistency with the current method.
Right now the users know there are classes that can be used to target specific series/dataindex, so it would be natural for them to try to do the same in all components.

Once we migrate away from them, users can change their codebase in one go.

Reply:
I suspect there will be little people that will be using this, so I'm ok with adding the classes.